scripts/deploy-sdks.ts

```diff
@@ -78,6 +78,27 @@ type Answers = {
     process.exit(1)
   }
 
+  let tag = 'latest'
+  const prerelease = semver.prerelease(version)
+  if (prerelease) {
+    tag = (prerelease[0] + '').trim()
+
+    if (!['alpha', 'beta', 'canary', 'rc'].includes(tag)) {
```

Comment:
🤔 Why do we need to check for those specific prerelease types? As in, what are the knowns prerelease keywords that would be ok to accept?

Reply:
honestly, purely an arbitrary choice. we don't have standards regarding these keywords, but they're pretty frequently used in the wild. We use only `alpha` and `beta` AFAIK